desk/app/profile/widgets.hoon

```diff
@@ -1,21 +1,23 @@
 ::  profile: construct stock widgets
 ::
-/-  co=contacts-0
+/-  co=contacts
 /+  sigil
 ::
 |=  =bowl:gall
-=/  ours=(unit contact-0:co)
-  =,  co
-  ::NOTE  we scry for the full rolodex, because we are not guaranteed to
-  ::      have an entry for ourselves, and contacts doesn't expose a "safe"
-  ::      (as in crashless) endpoint for checking
-  =+  .^  =rolodex
-        /gx/(scot %p our.bowl)/contacts/(scot %da now.bowl)/all/contact-rolodex
-      ==
-  =/  =foreign-0  (~(gut by rolodex) our.bowl *foreign-0)
-  ?:  ?=([[@ ^] *] foreign-0)
-    `con.for.foreign-0
-  ~
+=+  .^  =contact:co
+      /gx/(scot %p our.bowl)/contacts/(scot %da now.bowl)/v1/self/contact-1
+    ==
+::NOTE  can't quite make a nice helper for this, wetness not wet enough...
```

Comment:
This pattern seems like a good candidate to include as a safe, non-empty variant of `+get` in the `+cy` engine from `/lib/contacts`. Making a mental note to revisit this when I work on new %contacts features.

The code itself is good.